packages/cli/test/fixtures/repository/index.js

```diff
@@ -107,4 +108,11 @@ exports.SANDBOX_FILES = [
       encoding: 'utf-8',
     }),
   },
+  {
+    path: REPOSITORY_APP_PATH,
+    file: 'defaultmodel.repository.ts',
+    content: fs.readFileSync(require.resolve('./repositories/defaultmodel.repository.txt'), {
```

Comment:
Is there any particular reason why this fixture file has `.txt` extension instead of `.ts`?

Reply:
I created the file following the same naming convention used in the model folder, ex: ../fixtures/repository/models/decoratordefined.model.txt
You want me to change it?

Reply:
Yes please, let's use `.ts` or `.js` if possible.

Reply:
> Can we come up with a better solution? For example, we can use a different naming convention for generic repository classes (`my-crud-repository.base.ts`) or expect generic repositories to be defined in a different directory (e.g. `repositories/base`, `repositories/generic`, etc.).

I'll think about it in the next days... looks that the naming convention it's a good first step. I'll check that.

Reply:
@bajtos I added support for "base" repository files.
I used the file convention "{modelname}.repository.base.ts